AppCenterReactNativeShared/android/src/main/java/com/microsoft/appcenter/reactnative/shared/AppCenterReactNativeShared.java

```diff
@@ -75,11 +80,20 @@ public static String getAppSecret() {
                 configStream.close();
                 String jsonContents = new String(buffer, "UTF-8");
                 JSONObject json = new JSONObject(jsonContents);
-                AppCenterReactNativeShared.sAppSecret = json.getString("app_secret");
+                sAppSecret = json.getString("app_secret");
```

Comment:
Since our test helper code looks better than SDK code (:D), we should also define the keys as constants.